### DISCUSSION TITLE
Support for Xiaomi Vacuum 1G ( mijia.vacuum.v2 )

### DISCUSSION THREAD

**Discussion by edgano** [Hypfer/Valetudo#714] [Q&A / Support] [answered]
I would like if this device could be supported. It's the most simple xiaomi device nowadays (and cheap) and it would be interesting to be able to enjoy valetudo firm on it.

Thanks

**Comment by Hypfer** [comment, marked as answer]
Considering that it uses an ESP32, it is highly unlikely.

You could of course try dumping the CloudKey somehow and run Valetudo on another host, but considering that you could also get a proper vSLAM vacuum which can actually run Valetudo by itself (e.g. the Dreame F9) for ~30€ more, I really don't see the point in spending time on researching & implementing that.
